iOS/Flows/Conversation/Cells/EmotionGradientView.swift

```diff
@@ -59,7 +59,7 @@ class EmotionGradientView: BaseView {
 
         // Sort the colors by hue so more similar colors are near each other.
         colors = colors.sorted(by: { color1, color2 in
-            if color1.hue == color2.hue {
+            if abs(color1.hue - color2.hue) < 0.01 {
```

Comment:
The group similar colors together. I'm giving a little more wiggle room so we can handle floating point imprecision.